core/loadpoint.go

```diff
@@ -947,11 +958,29 @@ func (lp *LoadPoint) pvScalePhases(availablePower, minCurrent, maxCurrent float6
 	if !waiting && !lp.phaseTimer.IsZero() {
 		lp.log.DEBUG.Printf("phase timer reset")
 		lp.phaseTimer = time.Time{}
+
+		lp.publishTimer(time.Now(), 0, pvTimerInactive)
 	}
 
 	return false
 }
 
+func (lp *LoadPoint) publishTimer(timer time.Time, delay time.Duration, action string) {
+	remaining := delay - lp.clock.Since(timer)
+	if remaining < 0 {
+		remaining = 0
+	}
+
+	lp.publish("pvAction", action)
+	lp.publish("pvActionTimer", remaining)
```

Comment:
Wenn ich das richtig sehe werden hier nur zwei neue Felder (pvAction und pvActionTimer) bereitgestellt. Der Phase und PV Timer läuft ja gleichzeitig. Ich würde beide Timer bereitstellen. Vom Format denke ich an folgendes:

```
// pv
pvTimerRemaining: 100
pvTimerTotal: 600
pvTimerNextAction: enable|disable

// Phases
phaseTimerRemaining: 100
phaseTimerTotal: 600
phaseTimerNextAction: scaleUp|scaleDown

// timer inaktiv
pvTimerRemaining: null
pvTimerTotal: null
pvTimerNextAction: null
```

Reply:
Alternativ gerne auch als Unterobjekt je Timer:
```
timer: {
  pv: {total, remaining, nextAction}
  phase: {total, remaining, nextAction}
  pv: null // inaktiv
}
```

Reply:
ist umgesetzt- 2 Timer in 4 Werten. Total gibts nicht, aber Remaining.

Reply:
Meine Hoffnung wäre trotzdem das am Ende wieder zu konsolidieren um die Logik nicht im UI nachzubauen

Reply:
Wieso kann ich das `total` nicht kriegen? Ohne das kann man keine prozentualen Darstellungen machen.

Reply:
Fair point. Können wir es trotzdem erstmal ohne probieren? Lernen vor Perfektion?

Reply:
Problem ist dass die timer enable/disable unterschiedlich lang sind

Reply:
Naja, das schränkt die Darstellungsmöglichkeiten halt relativ ein. Irgend ein Timer läuft ja fast immer, würde gerne Flackern vermeiden, dafür wäre die Info ob der Timer gerade losgelaufen ist oder schon länger unterwegs ist (Bezugspunkt) halt total gut. Gibts denn ein technisches Problem warum du die nicht reinpacken kannst?

Reply:
Wenn du die willst sinds 3 timer. Ich wollte mit dem PR erstmal nur verstehen ob die überhaupt helfen. Schön machen können wir immer noch. Keine Logik, nur Anzeige!

Reply:
Bzw sind es dann 4 timer…